qbraid_algorithms/qtran/qasm_builder.py

```diff
@@ -133,7 +133,7 @@ class GateBuilder(FileBuilder):
     def import_library(self, lib_class, annotated=False):
         ret = super().import_library(lib_class, annotated)
         ret.call_space = " {}"
-        return  ret
+        return ret
```

Comment:
Not rendering correctly - 

<img width="1377" height="326" alt="Image" src="https://github.com/user-attachments/assets/b7a2629b-bdbd-4c55-9358-4050df14d57b" />